extensions/wikia/Flags/maintenance/RevertNotice.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+$dir = dirname( __FILE__ ) . "/../../../../";
+require_once( $dir . 'maintenance/Maintenance.php' );
+
+use Flags\Models\Flag;
+use Flags\Models\FlaggedPages;
+
+class RevertNotice extends Maintenance {
+
+	const EDIT_SUMMARY = 'Moving back templates from [[Special:Flags]] feature.';
```

Comment:
I'd phrase it as: "Restoring templates used in the [[Special:Flags]] feature" but @Grunny - please confirm. :)